cmd/agentbasedinstaller/host_config.go

```diff
@@ -192,10 +193,28 @@ func LoadHostConfigs(hostConfigDir string) (HostConfigs, error) {
 			}
 		}
 
-		configs = append(configs, &hostConfig{
-			configDir:    hostPath,
-			macAddresses: addresses,
-		})
+		addHostConfig := false
+		if workflowType == "addnodes" {
```

Comment:
For the sake of clarity, it could be worth redefining the workflow enum here, instead of using plain strings